docs/server.md

```diff
@@ -1,19 +1,26 @@
-Reactive Trader Cloud - Server
-===============
-
-* [Getting Started on Windows](./setup/windows-setup.md)
-* [Getting Started on Linux](./setup/linux-setup.md)
-* [Getting Started on Mac OS](./setup/macos-setup.md)
+# Reactive Trader Cloud - Server
```

Comment:
Do you think it would make sense to collapse the various getting started documents into one, and instead point people towards the setup instructions for the various tools that are environment-specific?

Because once you have installed crossbar, eventstore, dotnetcore etc. for your environment, the commands for running things up should be exactly the same regardless of environment.

This way we could de-duplicate the instructions across the environment-specific documents, and thereby prevent changes not being made everywhere at the same time in the future.